### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by makerboiAdi**
Added SuperTokens opportunity

### CODE REVIEW

**Comment by philipkiely** [comment]
Do you have a link to any sort of public "write for us" page e.g. https://auth0.com/guest-authors?

**Comment by malgamves** [comment]
thanks for the comment @philipkiely - do you have any update on that link @makerboiAdi ??

File: README.md [hunk 1]
```diff
@@ -115,6 +115,9 @@ A list of companies that have paid Developer Community Writer Programs.
 
 - [Strapi](https://strapi.io/write-for-the-community) - Up to $200 per piece
   > Articles or tutorials with code covering use-cases, solutions and projects built with Strapi that include Vue, Open Source, JavaScript, GraphQL, Jamstack, React. Pick from a list of possible articles or pitch your own.
+
+- [SuperTokens](https://angel.co/company/supertokens/jobs/771616-head-of-technical-content-and-documentation) - Upto $400 per article
+  > Articles on related topics (session management, authentication), tutorials with code covering use-cases and Product documentation
```
Comment (Qodestackr) [inline README.md:120]: This URL is broken
?

**Comment by karllhughes** [comment]
Looks like this was a temporary job opening, not a community writer program. @makerboiAdi feel free to make a new PR if they open a community writer application. Thanks!

**Pull request closed without merging**